### DISCUSSION TITLE
急！急求解决！关于hertzbeat自定义监控模版无法保存应用问题

### DISCUSSION THREAD

**Discussion by lixingtao6234** [apache/hertzbeat#1722] [Q&A]
急、急、急！
在hertzbeat监控模版，对已有的mysql监控模版进行编辑，编辑完成后保存报错，无法保存，请求的内容格式应该没有问题，我把监控模版里面一个监控模块name，复制下重新命名然后保存，仍然无法保存，急求解决。谢谢！

![1712646748373](https://github.com/dromara/hertzbeat/assets/165788161/1485e2f3-cba8-4323-b311-340b35a3a954)
[hertzbeat10-2024-04-09.0.log](https://github.com/dromara/hertzbeat/files/14914775/hertzbeat10-2024-04-09.0.log)
[hertzbeat10-2024-04-09-error.0.log](https://github.com/dromara/hertzbeat/files/14914776/hertzbeat10-2024-04-09-error.0.log)

**Comment by tomsun28** [comment]
hi，我们会尝试复现。你可以先直接修改后台YML模版文件来进行。
